gamestonk_terminal/portfolio/dg_api.py

```diff
@@ -1,62 +1,313 @@
-import degiroapi
+import gamestonk_terminal.config_terminal as config
 import pandas as pd
-import yfinance as yf
-from termcolor import colored
-from gamestonk_terminal.config_terminal import DG_USERNAME as user, DG_PASSWORD as pw
-from gamestonk_terminal.portfolio.portfolio_helpers import dg_positions_to_df
+import quotecast.helpers.pb_handler as pb_handler
 
-degiro = degiroapi.DeGiro()
+from quotecast.api import API as QuotecastAPI
+from trading.api import API as TradingAPI
+from trading.pb.trading_pb2 import (
+    Credentials,
+    LatestNews,
+    NewsByCompany,
+    ProductsInfo,
+    ProductSearch,
+    Update,
+)
+
+# SETUP CREDENTIALS
+credentials = Credentials(
+    int_account=None,
+    username=config.DG_USERNAME,
+    password=config.DG_PASSWORD,
+    one_time_password=config.DG_TOTP,
+    totp_secret_key=config.DG_TOTP_SECRET,
+)
+
+# SETUP TRADING API
+trading_api = TradingAPI(credentials=credentials)
+
+# DEFINE QUOTECAST API
+quotecast_api = None
+
+
+def fetch_additional_information(
+    positions: pd.DataFrame,
+) -> pd.DataFrame:
+    """Fetch extra information about the positions like :
+        - name
+        - isin
+        - symbol
+        - ...
+
+    Args:
```

Comment:
You're using wrong docstrings, we are using numpy doc. Should be along the lines:

```
Parameters
  ----------
  ticker : str
      Company's stock ticker

  Returns
  -------
  pd.DataFrame
      A DataFrame with financial info
```

Reply:
Same comment for all docstrings.

Reply:
Changed